src/main/webapp/components/utils/getField.js

```diff
@@ -7,7 +7,9 @@
  * @param defaultValue the value to be used if obj[field] === undefined.
  */
 export const getField = (obj, field, defaultValue) => {
-  return obj[field] !== undefined ?
+  return obj !== undefined ? 
+      obj[field] !== undefined ?
       obj[field] :
-      (defaultValue !== undefined ? defaultValue : null);
+      (defaultValue !== undefined ? defaultValue : null) :
```

Comment:
nit: I'd indent this nested ternary operator more than the one it's based in, for readability purposes

Reply:
Indented here 8ddf581